src/pages/HomePage/CreateAppModal.js

```diff
@@ -89,9 +89,9 @@ class CreateAppModal extends Component {
 			<Col span={4}>{records}</Col>
 			<Col span={4}>{calls}</Col>
 			<Col>
-				<Link to="/billings">
+				<a href="https://appbase.io/pricing/" target="_blank" rel="noopener noreferrer">
```

Comment:
@siddharthlatest are we not keeping the in-app billing pages?

Reply:
Since this link can appear before the app is created (think first app), the in-app view won't load. Also, we show FAQs and other rich info on the main pricing page, so I thought it would be better to open that instead.